library/src/main/java/com/google/android/exoplayer/extractor/webm/WebmExtractor.java

```diff
@@ -1212,6 +1215,14 @@ public void initializeOutput(ExtractorOutput output, int trackId, long durationU
           initializationData = hevcData.first;
           nalUnitLengthFieldLength = hevcData.second;
           break;
+        case CODEC_ID_FOURCC:
+            Pair<List<byte[]>, Long> fourccData = parseFourCCPrivate(
+                new ParsableByteArray(codecPrivate));
+            if (fourccData.second != FOURCC_COMPRESSION_VC1)
```

Comment:
To clarify, I was suggesting you could just throw ParserException in parseFourCCPrivate in the case that biCompression isn't the value you're looking for. Then this block just becomes:

```
mimeType = MimeTypes.VIDEO_VC1;
initializationData = parseFourCCPrivate(new ParsableByteArray(codecPrivate));
```

You could optionally rename the method to something like `parseFourCcVc1Private` if that makes it clearer under what conditions it's happy.